docs/pages/installation.mdx

```diff
@@ -56,9 +56,10 @@ server.
 
 Teleport maintains DEB and RPM package repositories for different operating
 systems, platforms, and Teleport versions. A server that installs Teleport from
-a DEB or RPM package must have systemd installed. You can also download TAR
-archives containing Teleport binaries. All installations include `teleport`,
-`teleport-update`, `tsh`, `tctl`, `fdpass-teleport` and `tbot`.
+a DEB or RPM package, or via `teleport-update`, must have systemd installed.
+You can also download TAR archives containing Teleport binaries. All installations
+include `teleport`,`teleport-update`, `tsh`, `tctl`, `fdpass-teleport` and `tbot`.
+Agent installations should use the one-line script or `teleport-update`.
```

Comment:
Should we mention `teleport-update` under "Recommended installation steps" instead? We haven't described the one-line script yet, so readers working straight through the Linux section won't know what this means.